frb_rust/Cargo.toml

```diff
@@ -31,15 +33,15 @@ console_error_panic_hook = { version = "0.1.7", optional = true }
 wasm-bindgen = { version = "0.2.81" }
 js-sys = { version = "0.3.60" }
 web-sys = { version = "0.3.58", features = [
-  "DedicatedWorkerGlobalScope",
-  "MessageEvent",
-  "MessagePort",
-  "ErrorEvent",
-  "Blob",
-  "BlobPropertyBag",
-  "Worker",
-  "Url",
-  "BroadcastChannel",
+	"DedicatedWorkerGlobalScope",
```

Comment:
nit: it would be great if the old format could be kept, because someone else in the future may also automatically change the format via IDE automatically, then the code blame history will be a mess... but that's not a super big problem.

Reply:
what toml formatter does the project use?

If it is [Taplo](https://taplo.tamasfe.dev), perhaps a configuration file may be helpful (https://taplo.tamasfe.dev/configuration/file.html)

Reply:
You are right, there is no format yet. I am just following flutter's convention: when there is no (auto) formatter, try to not change the code due to format.

However, feel free to configure a formatter (e.g. Taplo)! Just ensure that

- [ ] it is checked in CI (along with other formatters, like Dart and Rust formatter)
- [ ] it is run automatically when `just precommit` (trivial to do if you put is as a sibling to the existing Dart format etc)
- [ ] it has nice support and is popular, e.g. has VSCode and Intellij IDEA bulit-in support or plugin support